lib/mail/EmailAddress.php

```diff
@@ -74,7 +74,7 @@ public function setEmailAddress($emailAddress)
             filter_var($emailAddress, FILTER_VALIDATE_EMAIL))
         ) {
             throw new TypeException(
-                '$emailAddress must be valid and of type string.'
+                "{$emailAddress} must be valid and of type string."
```

Comment:
keep attention if a `$emailAddress` is not a string and it's can't be converto to string this generate a php error!
if you need to cast this value to string, probably the best choise is do something like: https://github.com/webmozart/assert/blob/master/src/Assert.php#L1145